lib/iex/lib/iex/autocomplete.ex

```diff
@@ -125,6 +128,51 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  defp get_signatures(name, module) when is_atom(module) do
+    with docs when is_list(docs) <- get_docs(module, [:function, :macro], name) do
+      Enum.map(docs, fn {_, _, [signature], _, _} -> signature end)
```

Comment:
Signature can actually be multiple lines according to the spec. So do this instead:

```suggestion
      Enum.map(docs, fn {_, _, signatures, _, _} -> Enum.join(signatures, " ") end)
```